presto-main/src/main/java/com/facebook/presto/operator/aggregation/sketch/theta/ThetaSketchStateFactory.java

```diff
@@ -25,6 +24,8 @@
 public class ThetaSketchStateFactory
         implements AccumulatorStateFactory<ThetaSketchAggregationState>
 {
+    private static final long SKETCH_INSTANCE_SIZE = ClassLayout.parseClass(Union.class).instanceSize();
```

Comment:
Union is an abstract class with no member variables--does it help give accurate results?  The data is found in a protected class `UnionImpl`.

Reply:
I'm aware that the actual impl is in the protected class. The result of `parseClass(..).instanceSize()` here returns 16 bytes, most due to class headers which puts it closer to the true size. The results in my comment above show a constant 294-byte deficit. We could reasonably leave it out as it doesn't make that much difference

Reply:
We can keep it, but can you memorialize it in a detailed comment explaining why this is safe to do?

Reply:
added